### DISCUSSION TITLE
Congratulation to the dev team / User Feedback

### DISCUSSION THREAD

**Discussion by nlsmart** [lxqt/lxqt#2156] [General]
Hi,

Congratulation to the dev team, LXQt 1.0.0 is working well on x86 and arm devices.

I appreciate the ISO date :-)
2021.12.15

I'm keen on the split function, please don't forget to add it on your web site, screenshots section, the availability to split with your lightweight file manager PCManFM. Split is a really practice feature.

![LXQt split 02](https://user-images.githubusercontent.com/50394853/146151794-4d9198d4-9848-4f57-8d9a-0e5e1a1a258a.png)

have a nice day.

**Comment by tsujan** [comment]
> ... don't forget to add it on your web site, screenshots section...

Good idea. Thanks!

**Reply by stefonarch** [reply]
https://github.com/lxqt/lxqt.github.io/pull/91

**Comment by stefonarch** [comment]
>don't forget to add it on your web site, screenshots section,

Will do, maybe another section with some app screenshots? Thanks for the feedback!

**Comment by nlsmart** [comment]
Happy new year.

Thanks for your feedback and your LXQt web site up date.
